wsd/Admin.cpp

```diff
@@ -1027,35 +1028,43 @@ void Admin::triggerMemoryCleanup(const size_t totalMem)
             memLimit << "% (" << static_cast<size_t>(_totalSysMemKb * memLimit / 100.) << " KB).");
 
     const double memToFreePercentage = (totalMem / static_cast<double>(_totalSysMemKb)) - memLimit / 100.;
-    int memToFreeKb = clamp<double>(memToFreePercentage * _totalSysMemKb, 0, std::numeric_limits<int>::max());
-    // Don't kill documents to save a KB or two.
-    if (memToFreeKb > 1024)
+    int64_t memToFreeKb =
+        clamp<double>(memToFreePercentage * _totalSysMemKb, 0, std::numeric_limits<int>::max());
```

Comment:
Quite a lot of fairly pointless churn here that took most of the time to review... hey-ho. If it could be split out into a pure re-factor commit in the 1st-case that just changes the indent that would be nice - with logic changes in a follow-on that would be good.